using-mobile-extensions/adobe-target/README.md

```diff
@@ -42,20 +42,26 @@ You can find your Target client code by going to the Setup tab and selecting Imp
 {% endtab %}
 
 {% tab title="iOS" %}
+Add the Target library to your project via your `Podfile` by adding `pod 'ACPTarget'`
 #### Objective-C
 
-1. Add the Target library to your project via your `Podfile` by adding `pod 'ACPTarget'`
-2. Import the Target and Identity library. 
+Import the Target and Identity library. 
 
    ```objectivec
-   #import <ACPCore_iOS/ACPCore_iOS.h>
-   #import <ACPTarget_iOS/ACPTarget_iOS.h>
-   #import <ACPIdentity_iOS/ACPIdentity_iOS.h>
+   #import "ACPCore.h"
+   #import "ACPTarget.h"
+   #import "ACPIdentity.h"
+   #import "ACPTargetRequestObject.h"
+   #import "ACPTargetPrefetchObject.h"
    ```
 
 #### Swift
 
-Update the bridging header
+   ```swift
+   #import ACPCore
+   #import ACPTarget
+   #import ACPIdentity
```

Comment:
You showed import for ACPTargetRequestObject and ACPTargetPrefetchObject in case of objective c and skipped it for swift. Any reason?